float32 voltage_v		# Servo rail voltage in volts
float32 rssi_v			# RSSI pin voltage in volts
